datasets/qasper/README.md

```diff
@@ -19,6 +19,55 @@ task_categories:
 task_ids:
 - closed-domain-qa
 paperswithcode_id: qasper
+train-eval-index:
+- config: qasper
```

Comment:
Hmm this dataset doesn't appear to be formatted as JSONLines, which is what we currently assume for question answering datasets on the Hub / AutoTrain 🤔 

I suggest excluding this one in the first pass

Reply:
sure! when you say 'exclude', does that mean that you don't want to add eval metadata? or just not run on it for now?

Reply:
Yep, I mean let's not put metadata on the datasets we don't have a predefined format for (yet). Otherwise, users will try to run eval jobs with bad defaults :)

Reply:
yup, excluded